### DISCUSSION TITLE
App Totally not working

### DISCUSSION THREAD

**Discussion by snhsadi** [Shabinder/SpotiFlyer#2790] [General]
The app has totally stopped working and showing failed to fetch link for each and every song! 
What happened?

**Comment by Shabinder** [comment]
Rewrite will be launched in a few weeks, so please stay tuned for that.

**Reply by ishanrai19** [reply]
!remindme in ?

**Reply by tedy02** [reply]
Saw you were rewriting it a while back. Excited to see we are close to seeing it! Is it still based on YouTube Music?

**Comment by sebbbbbb-cyber** [comment]
macOS says its damaged so its probably app side
